sonar-scanner/src/index.js

```diff
@@ -68,7 +70,7 @@ const action = async () => {
 
   if (waitForQualityGate) {
     // Wait for the quality gate status to update
-    const status = await core.group('Check Quality Gate', async () => checkQualityGate());
+    const status = await core.group('Check Quality Gate', async () => checkQualityGate(reportPath));
```

Comment:
I'd expect a change to the `checkQualityGate` implementation but I can't find it?

Reply:
It seems that it was implemented already, not sure why it wasn't exposed before (I guess it was just used for tests, but I think it works just fine like that too).